modules/op-tkn-trigger-management.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// *  cli_reference/tkn_cli/op-tkn-reference.adoc
+
+[id="op-tkn-trigger-management_{context}"]
+= Trigger management commands
+
+== tkn eventlistener
+Manage event listeners.
```

Comment:
I suggest to unify the style... there are `taskruns` and `pipelineruns` but `event listeners`, `trigger bindings` and `trigger templates` (without and with space)

Reply:
Have changed to the following style - TaskRun, Task, Pipeline, PipelineRun, TriggerBinding, TriggerTemplates, EventListeners

Reply:
@ppitonak @Preeticp  - I have changed the style of the entire doc as mentioned in the above comment. Please review and let me know wdyt. Thanks!!